velox/functions/prestosql/aggregates/SetAggregates.cpp

```diff
@@ -398,6 +398,11 @@ std::unique_ptr<exec::Aggregate> create(
       return std::make_unique<Aggregate<int32_t>>(resultType);
     case TypeKind::BIGINT:
       return std::make_unique<Aggregate<int64_t>>(resultType);
+    case TypeKind::HUGEINT:
+      if(inputType->isLongDecimal()) {
```

Comment:
You can add VELOX_CHECK(inputType->isLongDecimal()) before the if statement instead of the VELOX_NYI.